apps/vsce/src/components/engineService.ts

```diff
@@ -312,9 +317,24 @@ export class EngineService {
 		});
 	}
 
-	// @TODO implement
 	public async isCodemodEngineNodeLocated(): Promise<boolean> {
-		return Promise.resolve(Math.random() > 0.8);
+		const command = [
+			CODEMOD_ENGINE_NODE_COMMAND,
+			buildCrossplatformArg('--help'),
```

Comment:
why `--help`?

Reply:
we can use any command here, but probably --version is faster